spec/models/shf_application_spec.rb

```diff
@@ -338,7 +333,7 @@
           application.accept!
         end
         it 'assigns company email to application contact_email' do
-          expect(application.company.email).to eq application.contact_email
+          expect(application.companies.last.email).to eq application.contact_email
```

Comment:
Update the specification to accurately reflect the change, please.

Suggestion:
`it 'uses the email from the last company in the list for the application contact_email default' do`

Reply:
For now I changed it to:
```
"assigns app's latest-company email to application contact_email"
```
but I did not change this originally because of how I think the need to refactor the "app accepted" email - see discussion above.